index.js

```diff
@@ -40,6 +41,7 @@ module.exports = function createStore (reducer, initialState) {
     dispatch: dispatch,
     subscribe: subscribe,
     getState: getState,
-    replaceReducer: replaceReducer
+    replaceReducer: replaceReducer,
+    combineReducers: combineReducers
```

Comment:
@ahdinosaur oh, you're right. and instead of returning `createStore` the main file ought to return this object when required: https://github.com/reactjs/redux/blob/master/src/index.js#L28